test/decompiler/reference/engine/nav/navigate-h_REF.gc

```diff
@@ -591,7 +591,7 @@
 (defmethod should-display? nav-control ((obj nav-control))
   (and
    *display-nav-marks*
-   (nonzero? (logand (-> obj flags) (nav-control-flags bit0 display-marks)))
+   (nonzero? (logand (-> obj flags) (nav-control-flags display-marks bit0)))
```

Comment:
can you double check that it still does this? I thought I fixed it so `bit0` would come first, but I might have messed it up.

Reply:
nope it seems to come second